source/_layouts/sponsorship-tiers.html

```diff
@@ -0,0 +1,71 @@
+<div class="flex flex-col items-center justify-center space-y-8 lg:flex-row lg:items-stretch lg:space-x-8 lg:space-y-0">
+    <section class="flex flex-col w-full max-w-sm p-12 pt-2 space-y-6 bg-white rounded-lg shadow-md border-t-gray-400">
+        <!-- Name -->
+        <div class="flex-shrink-0">
+            <h3 class="text-4xl font-medium tracking-tight text-gray-500"
```

Comment:
It might be worth using `div` instead of `h3` to avoid adding an anchor for these headings (I guess it's not needed here), what do you think?